lib/openshift/cluster_operator.rb

```diff
@@ -0,0 +1,22 @@
+module BushSlicer
+  class ClusterOperator < ClusterResource
+    RESOURCE = "clusteroperators"
+    # return the an array of versions for a particular clusteroperator
+    def versions(user: nil, cached: true, quiet: false)
+      rr = raw_resource(user: user, cached: cached, quiet: quiet)
+      rr.dig('status', 'versions').map { |v| v['version'] }
+    end
+
+    def conditions(user: nil, cached: true, quiet: false)
+      rr = raw_resource(user: user, cached: cached, quiet: quiet)
+      rr.dig('status', 'conditions')
+    end
+
+    # returns a hash simliar to
+    # {"lastTransitionTime"=>2019-03-23 07:33:34 UTC, "status"=>"False", "type"=>"Progressing"}
+    def condition(type: type, user: nil, cached: true, quiet: false)
```

Comment:
looks like these methods can go to `Resource` but ok to be here for the time  being